schemas/extension-package-mobile.json

```diff
@@ -117,13 +113,33 @@
             },
             "artifact": {
               "$ref": "#/definitions/mavenFQArtifactName"
+            },
+            "coreArtifact": {
+              "$ref": "#/definitions/mavenFQArtifactName"
+            }
+          },
+          "additionalProperties": false,
+          "required": ["className", "artifact"]
+        },
+        {
+          "type": "object",
+          "properties": {
+            "className": {
+              "$ref": "#/definitions/javaClassName"
+            },
+            "artifact": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/mavenFQArtifactName"
```

Comment:
Shouldn't it be className that is an array? I don't think we want artifacts as an array